source/features/faster-pr-diff-options.tsx

```diff
@@ -107,6 +113,13 @@ function init(): false | void {
 		// Make space for the new button by removing "Changes from" #655
```

Comment:
This no longer works, can you update it?

Reply:
Not for me, anywhere:

<img width="499" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/117815546-cc255c00-b28f-11eb-9337-f052f7a04c6d.png">

Example URL: https://github.com/sindresorhus/refined-github/pull/4297/files/532f870c4b490c0b76c6c368d5bde6ffaff3465d..87445f7eb9ef94e7147249957b8e5c6ef88779aa